layout/_partial/sidebar-navigation.ejs

```diff
@@ -14,44 +14,38 @@
     </li>
 
     <!-- Homepage -->
-    <li id="sidebar-first-li">
-        <a href="<%= config.root %>" target="_self">
-            <i class="material-icons sidebar-material-icons">home</i>
-            <%= __('sidebar.homepage') %>
-        </a>
-    </li>
-
-    <!-- I'm Feeling Lucky -->
-    <!--
-    <li class="dropdown">
-        <a href="" target="_self">
-            <i class="material-icons sidebar-material-icons">explore</i>
-             <%= __("sidebar.imlucky") %>
-        </a>
-    </li>
-    -->
+    <% if(theme.sidebar.homepage === true) { %>
+            <li id="sidebar-first-li">
+                <a href="<%= config.root %>" target="_self">
+                    <i class="material-icons sidebar-material-icons"><%= theme.sidebar.homepage.icon %></i>
+                <%= __('sidebar.homepage') %>
+            </a>
+        </li>
+    <% } %>
 
     <!-- Archives  -->
-    <li class="dropdown">
-        <a href="#" class="ripple-effect dropdown-toggle" data-toggle="dropdown">
-            <i class="material-icons sidebar-material-icons">inbox</i>
-             <%= __('sidebar.archive') %>
-            <b class="caret"></b>
-        </a>
-        <ul class="dropdown-menu">
+    <% if(theme.sidebar.archives === true) { %>
+        <li class="dropdown">
+            <a href="#" class="ripple-effect dropdown-toggle" data-toggle="dropdown">
+                <i class="material-icons sidebar-material-icons"><%= theme.sidebar.archive.icon %></i>
```

Comment:
- It should be `theme.sidebar.archives.icon` (you forgot an 's')
- Please surround this line with the following condition: `if(theme.sidebar.archives.icon){`